extension/src/popup/views/SignTransaction.tsx

```diff
@@ -229,7 +245,9 @@ export const SignTransaction = () => {
     <El>
       <BackButton onClick={() => window.location.replace("/")} />
       <HeaderEl>Confirm Transaction</HeaderEl>
-      <SubheaderEl>{title} is requesting a transaction</SubheaderEl>
+      <SubheaderEl>
+        {title} from {url} is requesting a transaction
```

Comment:
Would def like to hear what others think, but if we think the url origin is enough here we could do
```
const u = new URL (url); u.origin;
```
 That should hopefully support all domain suffixes